bin/runtime/src/lib.rs

```diff
@@ -345,50 +335,89 @@ impl_opaque_keys! {
 }
 
 parameter_types! {
-    pub const Offset: u32 = 0;
+    pub const MillisecsPerBlock: u64 = DEFAULT_MILLISECS_PER_BLOCK;
+    pub const SessionPeriod: u32 = DEFAULT_SESSION_PERIOD;
+}
+
+impl pallet_elections::Config for Runtime {
+    type SessionPeriod = SessionPeriod;
```

Comment:
Why is this here and MillisecsPerBlock is not? It's probably fine, but I would like to understand myself...

Reply:
MillisecsPerBlock may be read of from pallet_timestamp but SessionPeriod cannot be read from pallet_session